remove duplicated SYS_ipc_subcall check ... further in the file, there
is another #if check to make sure the value is 418

--- linux/sh/syscallent.h
+++ linux/sh/syscallent.h
@@ -435,9 +435,6 @@
 	{ 5,	0,	printargs,		"SYS_398"	}, /* 398 */
 	{ 5,	0,	printargs,		"SYS_399"	}, /* 399 */
 
-#if SYS_ipc_subcall != 400
- #error fix me
-#endif
 	{ 8,	0,	printargs,		"socket_subcall"}, /* 400 */
 	{ 3,	TN,	sys_socket,		"socket"	}, /* 401 */
 	{ 3,	TN,	sys_bind,		"bind"		}, /* 402 */
